front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/ItineraryLayer.tsx

```diff
@@ -9,25 +9,29 @@ type ItineraryLayerProps = {
   layerOrder: number;
   geometry?: GeoJsonLineString;
   hideItineraryLine?: boolean;
+  showStdcmAssets: boolean;
 };
 
 export default function ItineraryLayer({
   layerOrder,
   geometry,
   hideItineraryLine = false,
+  showStdcmAssets,
```

Comment:
You can set it by default to false ;) This way, you only need to specify it in the 2 stdcm maps
```suggestion
  showStdcmAssets = false,
```

Reply:
This value is optional in the “Map” component, before being passed to the “ItineraryLayer” component. It is already set to “false” by default and specified only for Stdcm maps :)